app/views/web/dashboard/users/show.html.haml

```diff
@@ -51,7 +51,14 @@
         %ul
           - @test_task_assignments.each do |test_task_assignment|
             %li= test_task_assignment.test_task.title
-
+    - if @user.approver.present?
+      %tr
+        %td
+          %b= t('dashboard.users.table.approver')
+        %td
+          %ul.list-unstyled
+            %li= @user.approver.full_name
+            %li= mail_to @user.approver.email
```

Comment:
Let's just make a link to approvers profile.
`mail_to` and email are redundant here.